modules/apps/headless/headless-builder/headless-builder-test/src/testIntegration/java/com/liferay/headless/builder/resource/test/HeadlessBuilderResourceTest.java

```diff
@@ -3255,6 +3258,70 @@ private void _testGetOpenAPIInDifferentCompany() throws Exception {
 			Http.Method.GET);
 	}
 
+	private void _testPostInDifferentCompany(
+			String apiSchemaExternalReferenceCode,
+			ObjectDefinition objectDefinition)
+		throws Exception {
+
+		HTTPTestUtil.customize(
+		).withBaseURL(
+			"http://www.able.com:8080"
+		).withCredentials(
+			"test@able.com", "test"
+		).apply(
+			() -> {
+				assertSuccessfulJSONObject(
+					_createAPIEndpoint(
+						_API_ENDPOINT_ERC_2, Http.Method.POST,
+						_API_APPLICATION_PATH_1, null,
+						APIApplication.Endpoint.RetrieveType.SINGLE_ELEMENT.
+							getValue(),
+						APIApplication.Endpoint.Scope.COMPANY
+					).put(
+						"r_apiApplicationToAPIEndpoints_c_apiApplicationERC",
+						_API_APPLICATION_ERC_1
+					).toString(),
+					"headless-builder/endpoints", Http.Method.POST);
+				assertSuccessfulJSONObject(
+					null,
+					StringBundler.concat(
+						"headless-builder/schemas/by-external-reference-code/",
+						apiSchemaExternalReferenceCode,
+						"/requestAPISchemaToAPIEndpoints/",
+						_API_ENDPOINT_ERC_2),
+					Http.Method.PUT);
+				assertSuccessfulJSONObject(
+					null,
+					StringBundler.concat(
+						"headless-builder/schemas/by-external-reference-code/",
+						apiSchemaExternalReferenceCode,
+						"/responseAPISchemaToAPIEndpoints/",
+						_API_ENDPOINT_ERC_2),
+					Http.Method.PUT);
+
+				JSONObject jsonObject = JSONUtil.put(
+					"textProperty", RandomTestUtil.randomString());
+
+				String str = HTTPTestUtil.invokeToJSONObject(
```

Comment:
Maybe more elegant to keep the json Object and then, compare the results of both toString methods

Reply:
In fact, you could inline it too

Reply:
Well, I see you modified it already